Changes

```diff
@@ -127,11 +127,14 @@ Working version
    OCaml heap.
    (Jacques-Henri Jourdan, review by Damien Doligez)
 
-* #8691, #8897: Allocation functions are now guaranteed not to trigger any
-   OCaml callback when called from C. In long-running C functions,
-   this can be replaced with calls to caml_check_urgent_gc.
-   (Jacques-Henri Jourdan, review by Stephen Dolan, Gabriel Scherer and
-    Guillaume Munch-Maccagnoni)
+* #8691, #8897: Allocation functions are now guaranteed not to trigger
+   any OCaml callback when called from C. In long-running C functions,
+   this can be replaced with calls to caml_check_urgent_gc. Side
```

Comment:
Nitpick: maybe break the line before "Side", so that there are two paragraphs and a more readable structure?